src/hf_hydrodata/point.py

```diff
@@ -473,6 +495,12 @@ def get_site_variables(*args, **kwargs):
         Latitude range bounds for the geographic domain; lesser value is provided first.
     longitude_range : tuple, optional
         Longitude range bounds for the geographic domain; lesser value is provided first.
```

Comment:
It may be helpful for readability/maintanability to split some of the query calls in get_site_variables() into separate helper functions with doc strings describing what they do.

Reply:
I like this idea! I will add this to my backlog list for improvements to make to this package.